services/transport/graphql/types/mutation-payload-resolver.go

```diff
@@ -17,3 +17,16 @@ type CreateTenantPayloadResolverContract interface {
 	// Returns the provided clientMutationId as part of mutation request
 	ClientMutationId(ctx context.Context) *string
 }
+
+// UpdateTenantPayloadResolverContract declares the resolver that can returns the payload contains the result of updating an existing tenant
```

Comment:
can returns: "s" not required ;)

Reply:
Did not understand, what is it here that I need to change to "s"?